sympy/stats/joint_rv_types.py

```diff
@@ -633,8 +633,8 @@ def MultivariateEwens(syms, n, theta):
     ==========
 
     .. [1] https://en.wikipedia.org/wiki/Ewens%27s_sampling_formula
-    .. [2] https://www.researchgate.net/publication/280311472_The_Ubiquitous_Ewens_Sampling_Formula
-
+    .. [2] https://www.jstor.org/stable/24780825
+ 
```

Comment:
This line has trailing whitespace.

Reply:
I would still like the citation to list the author name, paper title etc rather than just being a URL.